qa/rpc-tests/test_framework.py

```diff
@@ -128,10 +128,14 @@ def main(self):
             print("Unexpected exception caught during testing: "+str(e))
             traceback.print_tb(sys.exc_info()[2])
 
-        if not self.options.nocleanup:
-            print("Cleaning up")
+        try:
             stop_nodes(self.nodes)
             wait_bitcoinds()
+        except:
+            pass
```

Comment:
Please report the exception w/ stack trace here (`traceback.print_exc()`). Ignoring exceptions, especially in tests, can cause useful diagnostic information to et lost.